internal/fxcontext/context_test.go

```diff
@@ -18,37 +18,60 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package fx
+package fxcontext
 
 import (
+	gcontext "context"
 	"testing"
 
-	"go.uber.org/fx/service"
-
 	"github.com/stretchr/testify/assert"
 
-	gcontext "context"
+	"go.uber.org/fx/service"
+	"go.uber.org/fx/ulog"
```

Comment:
Is this the import group ordering you use? Generally we use,

-   Standard library
-   Local package imports
-   Third party package imports